test/test_syscalls.c

```diff
@@ -149,8 +149,8 @@ static char *strings[5][3] = {
 	},
 };
 
-#define N_ITERATIONS		1000000
-static int counter;
+#define N_ITERATIONS		4000000
+volatile int counter;
```

Comment:
Meanwhile this could changed to unsigned, before some other tool tries to complain about possible signed overflow if `s()` is called a lot.

Reply:
Done